modules/hal_nordic/nrfx/nrfx_config_reserved_resources_ncs.h

```diff
@@ -0,0 +1,916 @@
+	     "PPI groups used by the IEEE802.15.4 radio driver overlap with those "
+	     "assigned to the MPSL.");
+
+BUILD_ASSERT((NRFX_DPPI136_CHANNELS_USED_BY_802154_DRV & NRFX_DPPI136_CHANNELS_USED_BY_MPSL) == 0,
+	     "PPI channels used by the IEEE802.15.4 radio driver overlap with those "
+	     "assigned to the MPSL.");
+
+BUILD_ASSERT((NRFX_DPPI136_GROUPS_USED_BY_802154_DRV & NRFX_DPPI136_GROUPS_USED_BY_MPSL) == 0,
+	     "PPI groups used by the IEEE802.15.4 radio driver overlap with those "
+	     "assigned to the MPSL.");
+
+BUILD_ASSERT((NRFX_PPI_CHANNELS_USED_BY_802154_DRV & NRFX_PPI_CHANNELS_USED_BY_MPSL) == 0,
+	     "PPI channels used by the IEEE802.15.4 radio driver overlap with those "
+	     "assigned to the MPSL.");
+
+BUILD_ASSERT((NRFX_PPI_GROUPS_USED_BY_802154_DRV & NRFX_PPI_GROUPS_USED_BY_MPSL) == 0,
+	     "PPI groups used by the IEEE802.15.4 radio driver overlap with those "
+	     "assigned to the MPSL.");
+
+BUILD_ASSERT((NRFX_PPIB_00_10_CHANNELS_USED_BY_802154_DRV &
+	      NRFX_PPIB_00_10_CHANNELS_USED_BY_MPSL) == 0,
+	     "PPI channels used by the IEEE802.15.4 radio driver overlap with those "
+	     "assigned to the MPSL.");
+
+BUILD_ASSERT((NRFX_PPIB_01_20_CHANNELS_USED_BY_802154_DRV &
+	      NRFX_PPIB_01_20_CHANNELS_USED_BY_MPSL) == 0,
+	     "PPI channels used by the IEEE802.15.4 radio driver overlap with those "
+	     "assigned to the MPSL.");
+
+BUILD_ASSERT((NRFX_PPIB_11_21_CHANNELS_USED_BY_802154_DRV &
+	      NRFX_PPIB_11_21_CHANNELS_USED_BY_MPSL) == 0,
+	     "PPI channels used by the IEEE802.15.4 radio driver overlap with those "
+	     "assigned to the MPSL.");
+
+BUILD_ASSERT((NRFX_PPIB_22_30_CHANNELS_USED_BY_802154_DRV &
+	      NRFX_PPIB_22_30_CHANNELS_USED_BY_MPSL) == 0,
+	     "PPI channels used by the IEEE802.15.4 radio driver overlap with those "
+	     "assigned to the MPSL.");
+
+BUILD_ASSERT((NRFX_PPIB_02_03_CHANNELS_USED_BY_802154_DRV &
+	      NRFX_PPIB_02_03_CHANNELS_USED_BY_MPSL) == 0,
+	     "PPI channels used by the IEEE802.15.4 radio driver overlap with those "
+	     "assigned to the MPSL.");
+
+BUILD_ASSERT((NRFX_PPIB_04_12_CHANNELS_USED_BY_802154_DRV &
+	      NRFX_PPIB_04_12_CHANNELS_USED_BY_MPSL) == 0,
+	     "PPI channels used by the IEEE802.15.4 radio driver overlap with those "
+	     "assigned to the MPSL.");
+
+BUILD_ASSERT((NRFX_PPIB_020_030_CHANNELS_USED_BY_802154_DRV &
+	      NRFX_PPIB_020_030_CHANNELS_USED_BY_MPSL) == 0,
+	     "PPI channels used by the IEEE802.15.4 radio driver overlap with those "
+	     "assigned to the MPSL.");
+
+#endif /* NRF_802154_VERIFY_PERIPHS_ALLOC_AGAINST_MPSL */
+
+/** @brief Bitmask that defines DPPI channels that are reserved for use outside
+ *         of the nrfx library.
+ */
+#define NRFX_DPPI0_CHANNELS_USED                                                                   \
+	(NRFX_DPPI0_CHANNELS_USED_BY_BT_CTLR | NRFX_DPPI0_CHANNELS_USED_BY_802154_DRV |            \
+	 NRFX_DPPI0_CHANNELS_USED_BY_MPSL)
```

Comment:
Just a question: Is handling of `CONFIG_NRFE` not needed anymore (in general, not in this particular place)?

Reply:
The NRFE reservations are empty for now. I decided to leave them out and talk to Magdalena afterwards.